Update dependencies

Comandeer opened the pull request.

KarolDawidziuk reviewed (changes requested):
Hi @Comandeer,

There are a few packages that we can bump, so please see my inline comments.
Also, as the `tslint` is deprecated we should check if there is a chance to use another one or discuss what we can do with it.

Comandeer commented on package.json:
```diff
@@ -30,45 +30,45 @@
 		"karma-firefox-launcher": "^2.1.2",
-		"karma-jasmine": "^4.0.1",
+		"karma-jasmine": "^4.0.2",
 		"karma-jasmine-html-reporter": "^1.7.0",
```
I've updated `jasmine-core` to v4.

Comandeer commented:
@KarolDawidziuk, thx for the review! I've updated the packages you mentioned, excluding `zone-js` (the version of Angular we're using requires `~0.11.4`). I've also switched CI from local Chrome to BrowserStack one and it seems that tests are passing again :tada: 

I haven't also made a switch from TSLint to ESLint as it requires changes in configuration (e.g. removing `tsconfig.base.json`).

KarolDawidziuk reviewed (approved):
LGTM! 🎉
